### DISCUSSION TITLE
Instrumentation for remote dev

### DISCUSSION THREAD

**Discussion by lukens** [quarkusio/quarkus#31372] [Q&A]
From #15319 it looks like instrumentation is disabled for remote dev.

Is that still the current status? Is this just something that is never going to work in remote dev?

Also, having spent some time trying to enable it in remote dev, would it be worth a note in the documentation saying that it can't be used in remote dev mode?

**Comment by maxandersen** [comment]
I can't remember why it's explicitly disabled but yes worth having note in documentation. 

@stuartwdouglas do you recollect why it was disabled ? Technical or security reason?

**Reply by stuartwdouglas** [reply]
I think technical. It would have needed additional code to make it work correctly, so it was easier to just disable it.
